source/extensions/filters/network/redis_proxy/command_splitter_impl.h

```diff
@@ -41,14 +42,15 @@ class CommandStats {
   Stats::Counter& total_;
   Stats::Counter& success_;
   Stats::Counter& error_;
+  Stats::Histogram& latency_;
```

Comment:
Can you potentially switch this over to using the stats macros so its consistent with everywhere else? Sorry I thought about asking this on the previous PR, but if we keep adding stats it's probably worth it to do so now.